src/main/java/model/Player.java

```diff
@@ -0,0 +1,28 @@
+package main.java.model;
+
+public class Player {
+	private String name;
+	private int cell;
+
+	public Player(String name) {
+		super();

```

Comment:
why super?

Reply:
autogenerated stuff got in. Would be mindful of this now